avogadro/qtplugins/crystal/importcrystaldialog.ui

```diff
@@ -0,0 +1,80 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<ui version="4.0">
+ <class>Avogadro::QtPlugins::ImportCrystalDialog</class>
+ <widget class="QDialog" name="Avogadro::QtPlugins::ImportCrystalDialog">
+  <property name="geometry">
+   <rect>
+    <x>0</x>
+    <y>0</y>
+    <width>400</width>
+    <height>300</height>
+   </rect>
+  </property>
+  <property name="windowTitle">
+   <string>Import Crystal</string>
+  </property>
+  <layout class="QGridLayout" name="gridLayout">
+   <item row="0" column="0">
+    <widget class="QLabel" name="label">
+     <property name="text">
+      <string>File extension for open babel conversion (default: POSCAR):</string>
```

Comment:
Is this using Open Babel, or Avogadro? Should use Open Babel capitalization, but it looks like the work above added a POSCAR format. It would be good to get a pointer to a file so I can see it in action.